docs/user-guide.rst

```diff
@@ -793,6 +803,13 @@ Cloud resources from an OIDC or SAML provider.
     https://cloud.google.com/iam/docs/workforce-identity-federation#workforce-pools-user-project
 
 
+Note that this library does not perform any validation on the token_url,
```

Comment:
Missing Security considerations header?

Reply:
No, this section already have header a few lines above